### DISCUSSION TITLE
RSS for blog with full content

### DISCUSSION THREAD

**Discussion by genebean** [thuliteio/doks#889] [Q&A] [answered]
Is there already a method for getting an RSS feed of the blog where the feed contains the full content of each post instead of just the description? Asking because I need this for use with Mailchimp so that I can auto-publish newsletters by posting to the blog. 

For what it's worth, I have this working on my site that am am porting over to this theme and am simply hoping I don't have to rebuild the rss templates in here like I did last time.

**Comment by h-enk** [comment, marked as answer]
The RSS feed is generated using a template ([Hugo internal](https://gohugo.io/templates/rss/) or custom). There no configuration on summary /content. So, you'd have to update /customize it for what you'd like.

I guess, replace `{{ .Summary | html }}` w/ `{{ .Description | html }}` should do the trick: https://github.com/h-enk/doks/blob/master/layouts/rss.xml#L23

**Reply by genebean** [reply]
Thanks. I figured this was the case, but wanted to make sure.

**Reply by genebean** [reply]
I have run `cp node_modules/@hyas/doks/layouts/rss.xml layouts/rss.xml` yet it seems nothing I do in my local `rss.xml` file is being reflected in http://localhost:1313/blog/index.xml ... any tips on what I am doing wrong or why this isn't working?

**Reply by h-enk** [reply]
Hmm, looks like a mounting conflict. 

Could you try adding `excludeFiles = "rss.xml"` to `./config/_default/module.toml` (line 22..24), like so:

```toml
[[mounts]]
  source = "node_modules/@hyas/doks/layouts"
  target = "layouts"
  excludeFiles = "rss.xml
```

**Reply by genebean** [reply]
Tried tonight, no change.

**Reply by genebean** [reply]
Here’s the branch I’m working in in case it helps. It does not include the attempted rss changes - those are still local. https://github.com/genebean/indiansprings-hugo/tree/isd-v2

**Reply by h-enk** [reply]
This should work (tested):

- ` cp node_modules/@hyas/doks/layouts/rss.xml layouts/blog/rss.xml`
- replace  `<description>{{ .Summary| html }}</description>` w/ `<description>{{ .Content | html }}</description>`

**Reply by genebean** [reply]
That works, thanks!

For anyone following along later, I actually took this a little further and added in the `xmlns:content="http://purl.org/rss/1.0/modules/content/"` namespace and used this for my feed:

```xml
      <description>
        {{ .Summary | html }}
      </description>
      <content:encoded>
        {{ .Content | html }}
      </content:encoded>
```
